doc/fonts_reference.md

```
@@ -0,0 +1,602 @@
+| Pattaya | Regular | ❌ No |
+| Sriracha | Regular | ❌ No |
+
+---
+
+## External Fonts
+
+### Directory Structure
+
+Organize fonts by family in subfolders:
+
+```
```

markdownlint-cli2 (0.18.1)

Fenced code blocks should have a language specified

(MD040, fenced-code-language)